CIScripts/Build/ProdCI.ps1

```diff
@@ -0,0 +1,92 @@
+. $PSScriptRoot\..\Common\DeferExcept.ps1
+. $PSScriptRoot\..\Build\Repository.ps1
+
+function Get-ProductionRepos {
+    Param ([Parameter(Mandatory = $true)] [string] $TriggeredProject,
+           [Parameter(Mandatory = $true)] [string] $TriggeredBranch,
+           [Parameter(Mandatory = $true)] [string] $GerritHost)
+
+    $ToolsBranch =      Get-GerritBranch -RepoName "contrail-build" `
```

Comment:
Get-GerritBranch is used for all repos at once every time, maybe consider reworking it to return map of repo -> branch for all repos at once? Not going to fight about it though.

Reply:
We discussed this with @jacek-izykowski but ultimately decided that symmetry with StagingCI.ps1 file is nicer if we go this route.